src/file-1.xoo

```diff
@@ -0,0 +1,5 @@
+// dummy line 1
+// dummy line 2
+// dummy line 3
+// dummy line 4
+// violate HasTag rule: xoo on line 5
```

Comment:
![Code Smell](https://sonarsource.github.io/sonarcloud-github-static-resources/v1/common/code_smell.png 'Code Smell') Code Smell: Has Tag ([xoo:HasTag](http://127.0.0.1:37429/organizations/org5/rules#rule_key=xoo%3AHasTag))

[See it in SonarCloud](http://127.0.0.1:37429/project/issues?id=sample&pullRequest=7006&issues=AWlr9th8M1H-wc5koAMw&open=AWlr9th8M1H-wc5koAMw)

[ISSUE_KEY]: AWlr9th8M1H-wc5koAMw